packages/cms-lib/lib/__tests__/config.js

```diff
@@ -942,14 +909,11 @@ describe('lib/config', () => {
           setConfig(CONFIG);
         });
 
-        it('supports both defaultPortal and defaultAccount', () => {
+        it('supports defaultAccount', () => {
```

Comment:
Can we add a test to cover backwards compatibility?

Reply:
I have one [here](https://github.com/HubSpot/hubspot-cms-tools/blob/8dd0aa1a45bddcab06b709655f7dd38cb6a9a56f/packages/cms-lib/lib/__tests__/config.js#L891), this was just removing the test case around supporting both at the same time